nix/build-dapp-package.nix

```diff
@@ -0,0 +1,88 @@
+{ pkgs }:
+
+let
+  remappings = xs:
+    builtins.foldl' pkgs.lib.mergeAttrs {}
+      (builtins.map
+        (x: {
+          "${x.name}/" = "${x}/dapp/${x.name}/src/";
+         } // x.remappings)
+         xs);
+  libPaths = xs:
+    builtins.foldl' pkgs.lib.mergeAttrs {}
+      (builtins.map
+        (x: {
+          "${x.name}" = "${x}/dapp/${x.name}/src";
+         } // x.libPaths)
+         xs);
+in
+  {
+    name
+  , src
+  , deps ? []
+  , extract ? true
```

Comment:
does this mean that it defaults to true? I don't think `extract` is commonly used

Reply:
Right. I took it from [here](https://github.com/dapphub/dapptools/pull/400/files#diff-a8c475bf1172056e5043d77688ed2e1bR20), but it makes sense that it should default to `false`, probably.